textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/field/TextField.java

```diff
@@ -0,0 +1,42 @@
+package edu.uci.ics.textdb.common.field;
+
+
+import edu.uci.ics.textdb.api.common.IField;
+
+/**
+ * Created by chenli on 3/31/16.
+ */
+public class TextField implements IField {

```

Comment:
I could not tell the difference of this field to `StringField`, do you want to clarify?